src/realm/error_codes.hpp

```diff
@@ -51,9 +55,21 @@ class ErrorCodes {
         WebSocket_InavalidExtension = 1010,
         WebSocket_InternalServerError = 1011,
         WebSocket_TLSHandshakeFailed = 1015, // Used by default WebSocket
+
+        /// WebSocket Errors - reported by server
+        WebSocket_Unauthorized = 4001,
+        WebSocket_Forbidden = 4002,
+        WebSocket_MovedPermanently = 4003,
+        WebSocket_Client_Too_Old = 4004,
+        WebSocket_Client_Too_New = 4005,
+        WebSocket_Protocol_Mismatch = 4006,
+        WebSocket_5xx_Server_Errors = 4007,
```

Comment:
See below - let's use `WebSocket_InternalServerError` for `WebSocket_5xx_Server_Errors`